src/commands/scanner/run/dfa.ts

```diff
@@ -43,29 +42,30 @@ export default class Dfa extends ScannerRunCommand {
 			summary: getMessage(BundleName.RunDfa, 'flags.targetSummary'),
 			description: getMessage(BundleName.RunDfa, 'flags.targetDescription'),
 			delimiter: ',',
+			default: '.',
```

Comment:
Ditto. We can't set this here (same reason as described in my previous comment).